runner/pool/pool.go

```diff
@@ -394,7 +394,11 @@ func (r *basePoolManager) AddRunner(ctx context.Context, poolID string) error {
 		return errors.Wrap(err, "fetching pool")
 	}
 
-	name := fmt.Sprintf("garm-%s", uuid.New())
+	prefix := pool.RunnerPrefix
+	if prefix == "" {
+		prefix = "garm"
+	}
+	name := fmt.Sprintf("%s-%s", prefix, shortid.MustGenerate())
```

Comment:
Love the short ID idea!

Reply:
.. I changed that a little bit, to avoid having '-' and '_' in the short id. However, I'm tempted to add some unicode smileys :)